deploy/examples/import-external-cluster.sh

```diff
@@ -14,6 +14,7 @@ MON_SECRET_CEPH_USERNAME_KEYNAME=ceph-username
 MON_SECRET_CEPH_SECRET_KEYNAME=ceph-secret
 MON_ENDPOINT_CONFIGMAP_NAME=rook-ceph-mon-endpoints
 ROOK_EXTERNAL_CLUSTER_NAME=$NAMESPACE
+ROOK_RBD_FEATURES=$ROOK_RBD_FEATURES
```

Comment:
```suggestion
ROOK_RBD_FEATURES=${ROOK_RBD_FEATURES:-"layering"}
```
I'd prefer we keep it this way, so it is backward compatible and we don't bother
users with having to setting this option explicitly.

Reply:
I *almost* did it that way, but had interpreted the request the way I had it.  Pushing this change.